### PR TITLE
Improve ECS errors outputs

### PR DESCRIPTION
**Pull request by igorkotof**
- Added ability to print out ECS task stopped reason after ECS deployment failure

Testing:
[asciicast](https://asciinema.org/a/ZkqkIBYY4SYQHhKzsxfMDIDd0)

### CODE REVIEW

**Review by DimmKirr — CHANGES_REQUESTED** [review]
Nice work! 
Let's add some ASCICinema to the PR to demo the functionality.

**Comment by igorkotof** [comment]
> Nice work!
> Let's add some ASCICinema to the PR to demo the functionality.

@AutomationD added testing video

File: ecs_deploy/cli.py [hunk 1]
```diff
@@ -589,10 +594,12 @@ def inspect_errors(service, failure_message, ignore_warnings, since, timeout):
     if timeout:
         error = True
         failure_message += ' due to timeout. Please see: ' \
-                           'https://github.com/fabfuel/ecs-deploy#timeout'
+                           'https://github.com/fabfuel/ecs-deploy#timeout \n'
         click.secho('')
 
     if error:
+        task_stopped_reason = action.get_task_stop_reason(service)
+        failure_message += 'Stopped reason of tasks:\n %s\n' % (task_stopped_reason)
```
Comment (DimmKirr) [inline ecs_deploy/cli.py:602]: Let's update message to something like:
> Stop reason of the task: 

Any objections?

**Review by DimmKirr — APPROVED** [review]
Amazing work @igorkotof, thank you!
Minor question about message language.

**Pull request merged**